pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarUtils.java

```diff
@@ -84,7 +84,7 @@ static StreamMessageMetadata extractMessageMetadata(Message<byte[]> message, Pul
     MessageIdStreamOffset nextOffset = new MessageIdStreamOffset(getNextMessageId(messageId));
     StreamMessageMetadata.Builder builder =
         new StreamMessageMetadata.Builder().setRecordIngestionTimeMs(recordIngestionTimeMs)
-            .setOffset(offset, nextOffset);
+            .setOffset(offset, nextOffset).setSerializedValueSize(message.size());
```

Comment:
null check for message

Reply:
The `message` variable is definetly not `null` because it is used several times because it is called in the code change.
```
  @VisibleForTesting
  static StreamMessageMetadata extractMessageMetadata(Message<byte[]> message, PulsarConfig config) {
    long recordIngestionTimeMs = message.getBrokerPublishTime().orElse(message.getPublishTime());
    MessageId messageId = message.getMessageId();
    MessageIdStreamOffset offset = new MessageIdStreamOffset(messageId);
    MessageIdStreamOffset nextOffset = new MessageIdStreamOffset(getNextMessageId(messageId));
    StreamMessageMetadata.Builder builder =
        new StreamMessageMetadata.Builder().setRecordIngestionTimeMs(recordIngestionTimeMs)
            .setOffset(offset, nextOffset).setSerializedValueSize(message.size());
```